epub33/a11y/index.html

```diff
@@ -915,83 +925,96 @@ <h6>Skippability</h6>
-									<p>When EPUB Creators add structural semantics to Media Overlay Documents, however,
-										Reading Systems can create reading experiences that allow users to decide which
-										secondary content to skip by default during playback.</p>
+										the same ease by default. Synchronized text-audio playback will typically result
+										in Reading Systems rendering secondary content where it occurs.</p>
+									<p>When EPUB Creators add structural semantics, however, Reading Systems can create
+										reading experiences that allow users to decide which secondary content to skip
+										by default during playback.</p>
 								</dd>
 
-								<dt id="sec-mo-skippability-conf">Meeting this Objective</dt>
+								<dt id="sec-sync-skippability-conf">Meeting this Objective</dt>
 								<dd>
-									<p>EPUB Creators SHOULD identify all <a
-											href="https://www.w3.org/TR/epub/#sec-skippability">skippable structures</a>
-										[[EPUB-3]] in Media Overlay Documents.</p>
+									<p>EPUB Creators SHOULD identify all skippable structures if the synchronized
+										text-audio playback format does not handle escaping content by default.</p>
+									<div class="note">
+										<p>Refer to <a data-cite="epub-a11y-tech-11#sync-003">Identifying skippable
+												structures</a> [[EPUB-A11Y-TECH-11]] for more information on meeting
+											this objective.</p>
+									</div>
 								</dd>
 							</dl>
 						</section>
 
-						<section id="sec-mo-escapability">
+						<section id="sec-sync-escapability">
 							<h6>Escapability</h6>
 
 							<dl>
-								<dt id="sec-mo-escapability-obj">Objective</dt>
+								<dt id="sec-sync-escapability-obj">Objective</dt>
 								<dd>
 									<p>Enable users to automatically escape from structured content.</p>
 								</dd>
 
-								<dt id="sec-mo-escapability-understand">Understanding this Objective</dt>
+								<dt id="sec-sync-escapability-understand">Understanding this Objective</dt>
 								<dd>
 									<p>When reading visually, users can quickly move through, and escape from, highly
 										structured content such as tables, lists, and figures. Table layouts that use
 										rows and columns, for example, allow users to quickly move along either axis to
 										find the information they want, and to easily return to the primary narrative
 										when they finish. Similarly, lists of items allow users to skim their content
 										and escape from them once they locate the desired information.</p>
-									<p>The same ease of escaping from content is not available in Media Overlay
-										Documents by default. Users cannot escape from table cells, rows, or even the
-										table itself, unless EPUB Creators encode the structural semantics of those
-										elements in the document.</p>
+									<p>The same ease of escaping from content is not always available in synchronized
```

Comment:
Replace with

"The same ease of escaping from content is only possible if EPUB Creators encode the structural semantics into the synchronized text/audio format."

Because at first glance it looks like we're saying that the features are not available in sync text/audio.